### ISSUE TITLE
Thr Lk Caption on PFD

### ISSUE THREAD

**Issue by JimMcl610** [flybywiresim/a32nx#6491]
### Aircraft Version

Experimental

### Build info

```json
{
  "built": "2022-01-06T07:18:33+00:00",
  "ref": "experimental",
  "sha": "1bb8d22c4cb3a52e7d6bb235e2398060497773e3",
  "actor": "aguther",
  "event_name": "manual"
}
```

### Describe the bug

Both when climbing and descending I have had random and unexplained THR LK flashing caption on the PFD and the A/THR has disconnected.  On one occasion at the BOD the A/THR re-engaged with no input from me.

### Expected behavior

None of the above.  A/THR should stay engaged.

### Steps to reproduce

Happens randomly but it has happened on the only 2 flights I have carried out on the Experimental version. Once in the decent and once in the climb. 2nd time A/THR never became available for the remainder of the flight.

### Discord Username (optional)

Specter#3708

**Comment by donstim** [comment]
Can you provide a video of this occurring. The TOGA LK condition follows the activation of Alpha Floor. See https://docs.flybywiresim.com/pilots-corner/advanced-guides/protections/afloor/?h=alpha#fma-indications. Alpha Floor is a protection mode to protect the airplane from stalling. If Alpha Floor activated, you were in danger of stalling the airplane due to a high angle-of-attack (usually low speed) condition. 

This should not normally occur with autopilot/autothrust active. However, spikes in MSFS 2020 live weather cause this too occur all too often.

**Comment by lukecologne** [comment]
The issue is talking about THR LK, not TOGA LK. When thrust lock is flashing on the PFD, it means that the autothrust was disengaged with the thrust levers in the CLB detent and is thus perfectly normal if the A/THR disengaged. The more interesting question would be why the A/THR disengaged in the first place. If you would upload an FDR file of a flight where this occured, it would be helpful.

**Comment by JimMcl610** [comment]
Where do I find the FDR file you refer to?  Key bindings are fine. Same as I have used with all previous versions and this started happening with the re-released Experimental branch.

**Comment by lukecologne** [comment]
FDR files are explained here: https://docs.flybywiresim.com/fbw-a32nx/support/?h=fdr#fdr-files

**Comment by aguther** [comment]
When you press ATHR disconnect buttons for more than 15 s the ATHR is disabled for the rest of the flight. Please ensure that your hardware is not generating this event by accident. This would also match to your first issue.

**Comment by JimMcl610** [comment]
Hello guys.
I did find a control binding that produced the symptoms I was having. A throw back to the ridiculous control binding menu MSFS uses and must be a remnant of a copied profile. I certainly didn't move that switch but that doesn't mean it isn't intermittently producing an output. I have unbound it and will continue to fly.  If the issue re-occurs I will send the FDR flight but I am quietly confident this has been caused by my control bindings and nothing you guys have done.
